### DISCUSSION TITLE
Refresh Backend Dataset

### DISCUSSION THREAD

**Discussion by tusharkudal** [ghiscoding/Angular-Slickgrid#1205] [General]
We are upgrading the slickgrid from version 3 to 5. Looks like extensionService.refreshBackendDataset() has been deprecated. Is there any replacement for this functionality or do I need to change the implementation?

**Comment by ghiscoding** [comment]
Have you tried it before posting this question? I don't remember deprecating it and from what I can see by looking at the code, it's still there, any important deprecations would have a note in the Migration Wikis (which I hope you are following). Please make sure to try it first and please consider asking these questions on Stack Overflow. 

Here's the method definition in Extension Service of Slickgrid-Universal
https://github.com/ghiscoding/slickgrid-universal/blob/master/packages/common/src/services/extension.service.ts#L335-L338

it's still used by the Grid Menu when used with a backend service
![image](https://github.com/ghiscoding/Angular-Slickgrid/assets/643976/e257863c-8cb7-46e0-ace1-d12131132c42)

**Comment by tusharkudal** [comment]
Hey, thank you for your response and sorry for taking up your time. Yes, I have been trying to resolve this issue for a long time now. I think it's because of upgrading all the libraries and angular version all together for which we need to change the approach now. BTW thanks for your response. I'll ask further questions on Stack Overflow

**Reply by ghiscoding** [reply]
As pointed out, make sure to follow [Migration Guide to 4.x](https://github.com/ghiscoding/angular-slickgrid/wiki/Migration-to-4.x) and also make sure to follow them in order, I create one for each major release and all code change are encapsulated in these migrations

**Reply by tusharkudal** [reply]
Sure, thank you!
